pom.xml

```diff
@@ -1445,6 +1437,44 @@
       </properties>
     </profile>
 
+    <!-- Requires hadoop.version to be 2.4.0 or later -->
+    <profile>
+      <id>java11</id>
```

Comment:
When we should call mvn with `-Pjava11`, is it possible to integration it into generate-artifact.sh to generate both `java8` and `java11`?

Reply:
I might recommend always build with java8, but feel free to run it with 11